docs/SetupDatabaseForAWS.md

```diff
@@ -1,30 +1,151 @@
+### Authentication method
+
+When you use Amazon Aurora, you must set `JDBC_URL`, `USERNAME`, and `PASSWORD` in the ScalarDB/ScalarDL properties file as follows.
+
+```properties
+scalar.db.contact_points=<JDBC_URL>
+scalar.db.username=<USERNAME>
+scalar.db.password=<PASSWORD>
+scalar.db.storage=jdbc
+```
+
+Please refer to the following document for more details on the properties for Amazon Aurora (JDBC databases).
+
+* [Getting Started with ScalarDB on JDBC databases](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-jdbc.md)
+
+### Required configuration/steps
+
+#### Create an Amazon Aurora DB cluster
+
+You must create an Amazon Aurora DB cluster. Please refer to the official document for more details.
+
+* [Configuring your Amazon Aurora DB cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_AuroraSettingUp.html)
+
+### Optional configurations/steps
+
+#### Configure backup configurations (Optional based on your environment)
+
+Amazon Aurora automatically gets a backup by default. You do not need to enable the backup feature manually.
+
+If you want to change some backup configurations like the backup retention period and backup window, you can configure them. Please refer to the official document for more details.
+
+* [Backing up and restoring an Amazon Aurora DB cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/BackupRestoreAurora.html)
+
+#### Configure monitoring (Recommended in the production environment)
+
+You can configure the monitoring and logging of Amazon Aurora using its native feature. Please refer to the official documents for more details.
+
+* [Monitoring metrics in an Amazon Aurora cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/MonitoringAurora.html)
+* [Monitoring events, logs, and streams in an Amazon Aurora DB cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_Monitor_Logs_Events.html)
+
+#### Disable public access (Recommended in the production environment)
+
+Public access is disabled by default. You can access the Amazon Aurora DB cluster from the Scalar product pods on your EKS cluster as follows.
 
-* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create Amazon Aurora MySQL or Amazon Aurora PostgreSQL.
-* (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/MonitoringOverview.html), monitoring is enabled on the Amazon Aurora cluster by default.
\ No newline at end of file
+* Create the Amazon Aurora DB cluster on the same VPC as your EKS cluster.
+* Connect the VPC for the Amazon Aurora DB cluster and the VPC for the EKS cluster for the Scalar product deployment using [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html). (// TODO: We need to test this feature with Scalar products.)
```

Comment:
We have not yet tested this feature with Scalar products.
I will do the test and update the document as another task.